dotnet/G3DViewer/MainWindow.xaml.cs

```diff
@@ -111,47 +131,42 @@ public MainWindow()
                 attributeStat.DataArity = attribute.Descriptor._data_arity;
                 attributeStat.DataType = ((DataType)attribute.Descriptor._data_type).ToString();
 
-
                 mDisplayStats.AttributeStats.Add(attributeStat);
             }
 
-            var groups = Ara3D.Revit.DataModel.RevitDataModelExtensions.SplitByGroup(g3d);
-                
-            /*
-            if (false) // instancing
-            {
-                foreach (var group in groups)
-                {
-                    mainViewModel.AddG3DData(group);
-                }
+            IArray<int> materialIds = G3DExtensions.MaterialIds(mG3D);

```

Comment:
`var` (just get into the habit of using it everywhere). There are tons of places, where I haven't bothered saying it. ReSharper will replace it throughout your project for you automatically.